bio/prinseq-plus-plus/wrapper.py

```
@@ -0,0 +1,55 @@
+# Input format
+in_fmt = Path(snakemake.input[0].removesuffix(".gz")).suffix
+if in_fmt in ext_fq:
+    pass
+elif in_fmt in ext_fas:
+    extra += " -FASTA"
+else:
+    raise ValueError("Invalid input file format")
+
+
+# Output files
+for key, value in snakemake.output.items():
```

Ruff

Undefined name `snakemake`

(F821)